hack/build-marker-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
please use `#!/usr/bin/env bash` (all over)
it is indeed less compact, but portable

Reply:
Done.